frontend/src/features/projects/common/ApprovalTransitionPage.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { Container, Spinner } from 'react-bootstrap';
+import { useProject, updateWorkflowStatus, IProject, IProjectWrapper } from '.';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from 'reducers/rootReducer';
+import _ from 'lodash';
+import { useHistory } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { fetchProjectWorkflow } from './projectsActionCreator';
+
+interface IApprovalTransitionPageProps {}
+
+const transitionFunction = (promise: any, history: any, toStatusCode: string) => {
+  return promise
+    .then((project: IProject) => {
+      if (project?.status?.route === undefined) {
+        throw Error('No valid route for current project status');
+      }
+      history.replace(`${project?.status?.route}?projectNumber=${project.projectNumber}`);
+    })
+    .catch(() => {
+      throw Error(`Failed to update status to ${toStatusCode}`);
+    });
+};
+
+const ApprovalTransitionPage: React.FunctionComponent<IApprovalTransitionPageProps> = props => {
```

Comment:
This page handles transitioning between statuses automatically when a status only has one valid toStatus. Made this a page to try and allow for re-use whenever this situation arises (currently just twice). The user won't be able to navigate to this page with the back/forward buttons due to the use of history.replace.